### ISSUE TITLE
rebind keys to shoulder buttons 

### ISSUE THREAD

**Issue by mastercheff** [fgsfdsfgs/doukutsupsx#3]
Any chance of inplementing weapon switch on L1&R1 L2&R2 ? cheers

**Comment by fgsfdsfgs** [comment]
Yeah, that can be done, just didn't see a need to do it. Would require button images for L1/L2/R1/R2 though.
